features/steps/steps.rb

```diff
@@ -67,3 +67,22 @@ def assert_payload_fields_match(field, expected, actual)
     "Expected the last command to exit successfully, but it exited with code #{Docker.last_exit_code}"
   )
 end
+
+Then("the exit code is not successful") do
+  assert_not_equal(
+    Docker.last_exit_code,
+    0,
+    "Expected the last command to exit unsuccessfully, but it exited with code 0"
+  )
+end
+
+Then('the shell has output {string} to stdout') do |expected_line|
+  assert(
+    Docker.output.any? { |line| line.include?(expected_line) },
```

Comment:
The actual MR step for this does an exact match, which doesn't quite work as it's sensitive to whitespace and ANSI escape codes

We also may want a regex version so we can assert against the success case, which outputs:

> ✔ Success, uploaded dist/main.js.map to http://maze-runner:9339 in 23ms

Where "23ms" will change every run